core-bundle/contao/drivers/DC_Table.php

```diff
@@ -2698,9 +2698,11 @@ public function editAll($intId=null, $ajaxId=null)
 						continue;
 					}
 
-					$return .= Message::generateUnwrapped() . '
-<div class="' . $class . ' cf">' . $box . '
-</div>';
+					$return .= Message::generateUnwrapped() . \sprintf('<fieldset data-controller="contao--toggle-fieldset" data-contao--toggle-fieldset-collapsed-class="collapsed" class="%s cf"><legend><button type="button" data-action="contao--toggle-fieldset#toggle">%s</button></legend>%s</fieldset>',
+						$class,
+						StringUtil::specialchars(System::getContainer()->get('contao.data_container.record_labeler')->getLabel('contao.db.' . $this->strTable . '.id', $currentRecord)),
```

Comment:
```suggestion
						StringUtil::specialchars(System::getContainer()->get('contao.data_container.record_labeler')->getLabel('contao.db.' . $this->strTable . '.' . $currentRecord['id'], $currentRecord)),
```

Note to myself: `contao.db.tl_foo.id` should not be handled by the default listener, only valid identifiers like `contao.db.tl_foo.123`

Reply:
Oh, should I change? :D